### PR TITLE
Logit to paddle tensor.py

### PR DESCRIPTION
**Pull request by AkshayVenkataraman**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -2328,3 +2328,38 @@ def test_paddle_logical_not(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# logit
+@handle_frontend_method(
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:2334]: `handle_frontend_method` is not defined

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 2]
```diff
@@ -2328,3 +2328,38 @@ def test_paddle_logical_not(
...
+    class_tree=CLASS_TREE,
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:2335]: `CLASS_TREE` is not defined

**Review by kurshakuz — CHANGES_REQUESTED** [review]
Hey! Your implementation is not working currently, so please fix them before next review, thanks!

**Pull request closed without merging**